crates/autopilot/src/domain/surplus_capturing_jit_order_owners.rs

```diff
@@ -0,0 +1,37 @@
+use {primitive_types::H160, std::collections::HashSet};
+
+/// Surplus capturing JIT order owners
+/// The list of owners is initialized with the values specified in the autopilot
+/// configuration file, and it is extended with all the CoW AMM owners which are
+/// deployed by the CoW AMM helper contract
+pub struct SurplusCapturingJitOrderOwners {
```

Comment:
We actually don't necessarily need this, but I thought it would be nice to have a domain structure containing all these methods.

Reply:
If you decide to keep this struct, I would at least move it to `autopilot/domain/auction/` since the concept is not important enough to be in the domain root.